board.py

```diff
@@ -112,141 +109,140 @@ def getTilePositions(self, position):
                 print("0 orientationCode xCoord yCoord")
                 print("Example: 0 5 A 2")
                 return False
-            return self.insertCard(args)
+            return self.insert_card(args)
         else:
             if len(args) != 7:
                 print("Input error: Swap moves should have 7 arguments (not " + str(len(args)) + "):")
                 print("xCoordTile1 yCoordTile1 xCoordTile2 yCoordTile2 newOrientationCode newCoordX newCoordY")
                 print("Example: F 2 F 3 3 A 2")
                 return False
-            return self.swapCard(args)
+            return self.swap_card(args)
 
-    def swapCard(self, args):
+    def swap_card(self, args):
         # TODO: implement this method
-        raise Exception("Method not implemented yet (lol)")
+        pass
 
-    def insertCard(self, inputArgs):
+    def insert_card(self, input_args):
         """ Tries to insert card into the board from the inputArgs given by player.
             If it was successful in inserting it, it increments self.nbrCards by 1 and returns the newCard.
             Otherwise, it returns None.
         """
         # Ensure that an exception is thrown when we try to exceed the max number of cards to indicate failure
         if self.nbrCards >= self.maxNbrCards:
-            print("Error: Cannot insert more than " + self.maxNbrCards + " cards on the board.\n"\
-                  "Please do a recycling move instead.")
+            print("Error: Cannot insert more than " + self.maxNbrCards + " cards on the board.\nPlease do a recycling "
+                                                                         "move instead.")
         try:
-            inputRotCode = int(inputArgs[1])
+            input_rot_code = int(input_args[1])
         except ValueError:
-            print("The 2nd argument should be an integer and " + inputArgs[1] + " is not.")
+            print("The 2nd argument should be an integer and " + input_args[1] + " is not.")
             return None
-        if inputRotCode <= 0 or inputRotCode > Card.NBR_ROTATION_CODES:
-            print("Valid rotation codes range from 1 to " + str(Card.NBR_ROTATION_CODES) + ", " +\
-                  "thus " + str(inputRotCode) + " is out of range.")
+        if input_rot_code <= 0 or input_rot_code > Card.NBR_ROTATION_CODES:
+            print("Valid rotation codes range from 1 to " + str(Card.NBR_ROTATION_CODES) + ", " + "thus " +
+                  str(input_rot_code) + " is out of range.")
             return None
 
-        newCard = Card(inputRotCode)
+        new_card = Card(input_rot_code)
         try:
-            positionNewCard = self.convertCoordinate((inputArgs[2], int(inputArgs[3])))
-        except Exception:
-            print(inputArgs[2] + " " + inputArgs[3] + " does not represent a valid position.")
+            position_new_card = self.convert_coordinate((input_args[2], int(input_args[3])))
+        except OutOfBoundsException as e:
```

Comment:
The hardcoded checks are not necessary, because an exception will be thrown here if we are out of bounds of the 2d array.